py4DSTEM/visualize/vis_special.py

```diff
@@ -44,31 +47,29 @@ def show_elliptical_fit(ar,center,Ri,Ro,a,e,theta,fill=True,
         return fig,ax
 
 
-def show_amorphous_ring_fit(dp,qmin,qmax,p_ellipse,N=12,cmap=('gray','gray'),
+def show_amorphous_ring_fit(dp,fitradii,p_ellipse,N=12,cmap=('gray','gray'),
                             fitborder=True,fitbordercolor='k',fitborderlw=0.5,
                             scaling='log',ellipse=False,ellipse_color='r',
                             ellipse_alpha=0.7,ellipse_lw=2,returnfig=False,**kwargs):
     """
     Display a diffraction pattern with a fit to its amorphous ring, interleaving
     the data and the fit in a pinwheel pattern.
 
-    Accepts:
-        dp              (array) the diffraction pattern
-        qmin,qmax       (numbers) the min/max distances of the fitting annulus
-        p_ellipse       (11-tuple) the fit parameters to the double-sided gaussian
-                        fit function returned by fit_ellipse_amorphous_ring
-        N               (int) the number of pinwheel sections
-        cmap            (colormap or 2-tuple of colormaps) if passed a single cmap,
-                        uses this colormap for both the data and the fit; if passed
-                        a 2-tuple of cmaps, uses the first for the data and the
-                        second for the fit
-        fitborder       (bool) if True, plots a border line around the fit data
-        fitbordercolor  (color) color of the fitborder
-        fitborderlw     (number) linewidth of the fitborder
-        scaling         (str) the normal scaling param -- see docstring for
-                        visualize.show
-        ellipse         (bool) if True, overlay an ellipse
-        returnfig       (bool) if True, returns the figure
+    Args:
+        dp (array): the diffraction pattern
+        fitradii (2-tuple of numbers): the min/max distances of the fitting annulus
+        p_ellipse (11-tuple): the fit parameters to the double-sided gaussian fit
+            function returned by fit_ellipse_amorphous_ring
+        N (int): the number of pinwheel sections
+        cmap (colormap or 2-tuple of colormaps): if passed a single cmap, uses this
+            colormap for both the data and the fit; if passed a 2-tuple of cmaps, uses
+            the first for the data and the second for the fit
+        fitborder (bool): if True, plots a border line around the fit data
+        fitbordercolor (color): color of the fitborder
+        fitborderlw (number): linewidth of the fitborder
+        scaling (str): the normal scaling param -- see docstring for visualize.show
+        ellipse (bool): if True, overlay an ellipse
```

Comment:
ellipse_alpha, _lw, and _color arguments are not defined

Reply:
same comment as above.....these do need updating/documenting.....